keps/sig-cluster-lifecycle/draft-20180130-kubeadm-join-master.md

```diff
@@ -280,7 +280,7 @@ existing `kubeadm join` flow:
       in `kube-system` namespace.
 
       > This requires to grant access to the above configMap for
-      `system:node-bootstrapper` group (or to provide the same information
+      `system:node-bootstrapper` ClusterRole (or to provide the same information
```

Comment:
You can't grant a role permission and you shouldn't modify system roles. I think this sentence has to change a little bit more.

Reply:
What about change `system:node-bootstraper` to `system:bootstrappers`? `system:bootstrappers` is the group of `system:bootstrap:<token-id>` user.

Reply:
That works.

Reply:
ok, I changed that in a new commit.